function-images/lr_serving/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apk add --no-cache \
     pip install --disable-pip-version-check --no-build-isolation pandas==1.0.5 && \
     pip install --disable-pip-version-check --no-build-isolation protobuf==3.11.3 && \
     pip install --disable-pip-version-check --no-build-isolation grpcio==1.26.0 && \
+    pip install --disable-pip-version-check --no-build-isolation pybind11 && \
```

Comment:
can you please explain the fix?

Reply:
[Scipy/fft](https://github.com/scipy/scipy/blob/v1.6.0/scipy/fft/_pocketfft/setup.py) build failed due to missing pybind11 dependency.